resources/leiningen/new/kraken/queue.clj

```diff
@@ -24,9 +25,19 @@
           (do (Thread/sleep (* attempt 1000))
               (recur (inc attempt)))
           (do (log/error "Connecting to RabbitMQ failed. Bailing.")
-              (throw (ex-info "Connecting to RabbitMQ failed" {:attemts attempt})))))))
-  (reset! channel
-          (let [ch (lch/open @connection)]
-            (lq/declare ch (config :rabbit-mq :queue) {:exclusive false :auto-delete false :durable true})
-            (log/info "RabbitMQ channel declared.")
-                        ch)))
+              (throw (ex-info "Connecting to RabbitMQ failed" {:attemts attempt}))))))
+    (let [ok-ch (lch/open @connection)]
+      (lq/declare ok-ch
+                  "{{name}}.ok"
+                  (config :rabbitmq :queues "{{name}}.ok"))
+      (k/responder ok-ch "{{name}}.ok" handlers/ok)
+      {:connections [@connection]
+       :channels [ok-ch]})))
```

Comment:
Why vectors over sets? Is order ever significant (now that we separated channels from connections)?